spec/features/browsing_item_spec.rb

```diff
@@ -8,19 +8,19 @@
   end
 
   it 'renders an accessible icon for citing the item' do
-    visit 'catalog/3478898'
+    visit 'catalog/9934788983506421'
     expect(page).to have_selector '.icon-cite[aria-hidden="true"]', visible: false
   end
 
   it 'renders an accessible icon for sending items to a printer' do
-    visit 'catalog/3478898'
+    visit 'catalog/9934788983506421'
     expect(page).to have_selector '.icon-share[aria-hidden="true"]', visible: false
     expect(page).to have_selector '.icon-print[aria-hidden="true"]', visible: false
   end
 
   context 'when an entry has a bib. ID for a resource published in Figgy' do
     before do
-      visit 'catalog/3753928'
+      visit 'catalog/9970446223506421'
```

Comment:
Question: Why not `9937539283506421`?

Reply:
It tests for thumbnail and UV. I used the example with the 2 viewers. https://catalog.princeton.edu/catalog/9970446223506421https://figgy.princeton.edu/catalog?utf8=%E2%9C%93&q=9970446223506421 it is in figgy.